debian/control

```diff
@@ -78,7 +78,8 @@ Vcs-Browser: https://salsa.debian.org/gnome-team/gnome-shell
 
 Package: gnome-shell
 Architecture: linux-any
-Depends: evolution-data-server (>= 3.33.1),
+Depends: adwaita-icon-theme,
```

Comment:
Is this needed? `mutter`, `libgtk-3-0`, `eos-icon-theme` all depend on it. The latter is pulled in by `eos-meta`.